dashboard/app/models/pd/enrollment_notification.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: pd_enrollment_notifications
+#
+#  id               :integer          not null, primary key
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  pd_enrollment_id :integer          not null
+#  name             :string(255)
+#
+# Indexes
+#
+#  index_pd_enrollment_notifications_on_pd_enrollment_id  (pd_enrollment_id)
+#
+
+class Pd::EnrollmentNotification < ActiveRecord::Base
+  belongs_to :enrollment, class_name: 'Pd::Enrollment', foreign_key: :pd_enrollment_id
+end
```

Comment:
Personally, I think saving three characters is not a sufficiently good reason to override the rails default (even if the context is "always clear"). Overriding introduces extra cognitive load, requiring us to know that the model `enrollment` corresponds to the DB `pd_enrollment` and vice versa.